src/TemplateStudio/index.js

```diff
@@ -25,6 +25,8 @@ const TLWrapper = styled.div`
   }
 `;
 
+/* eslint react/prop-types: 0 */
+/* This disables ESLint because Redux is handling props */
```

Comment:
you should be able to still use propTypes for the props coming from redux. let me know if it's not working for some reason and we can look at it together